polkadot/node/core/dispute-coordinator/src/tests.rs

```diff
@@ -2968,17 +2981,23 @@ fn participation_with_offchain_disabling() {
 			// let's disable validators 3, 4 on chain, but this should not affect this import
 			let disabled_validators = vec![ValidatorIndex(3), ValidatorIndex(4)];
 			handle_disabled_validators_queries(&mut virtual_overseer, disabled_validators).await;
```

Comment:
I am confused. We are disabling validators 3 and 4, but the votes are coming from 1 and 2? Ok, so this should not affect the import, make sense, but why should it affect participation further down?